idaes/apps/grid_integration/examples/thermal_generator_prescient_plugin.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+import pyomo.environ as pyo
+from pyomo.common.config import ConfigDict, ConfigValue
+from idaes.apps.grid_integration import Tracker
+from idaes.apps.grid_integration import Bidder
+from idaes.apps.grid_integration import PlaceHolderForecaster
+from idaes.apps.grid_integration import DoubleLoopCoordinator
+from thermal_generator import ThermalGenerator
```

Comment:
If this refers to the `thermal_generator.py` file in the same directory, it should work using a relative import:

```suggestion
from .thermal_generator import ThermalGenerator
```

Reply:
Thanks! I fixed it and will push later.